intelmq/bots/BOTS

```diff
@@ -421,6 +421,18 @@
                 "rate_limit": 180000
             }
         },
+        "MISP Generic": {
+            "description": "Collect events from a MISP server.",
+            "module": "intelmq.bots.collectors.misp.collector",
+            "parameters": {
+                "feed": "misp_generic",
+                "misp_url": "<insert url of MISP server (with trailing '/')>",
```

Comment:
The alphabetic order is: misp_key, misp_tag_processed, misp_tag_to_process, misp_url